substrate/frame/support/src/migrations.rs

```diff
@@ -504,6 +504,11 @@ pub trait SteppedMigration {
 		None
 	}
 
+	/// The prefixes that are being migrated.
```

Comment:
Some more context please. Imagine you are a developer and this is the only thing you know about the function.

```suggestion
	/// The prefixes that are being migrated.
	///
	/// This function is optional and provides information to the migration framework to know which storage prefixes are being migrated. It can be helpful to let chain explorers know which part of the state is possibly in a state where it cannot be read correctly.
```